quizapp/src/test/groovy/billennium/quizapp/CandidateITSpec.groovy

```diff
@@ -22,10 +28,19 @@ class CandidateITSpec extends Specification {
     @Autowired
     private CandidateRepository candidateRepository;
 
+    @Autowired
+    private QuizDefinitionRepository quizDefinitionRepository;
+
     def setup() {
         candidateRepository.save(Candidate.builder()
                 .email("billenet@billennium.com")
                 .build())
+
+        quizDefinitionRepository.save(QuizDefinition.builder()
+                .title("BIG-DATA")
+                .questions(new ArrayList<Question>())
```

Comment:
Jak możesz to poszukaj 'ladniejszych'  wydajeniejszych deklaracji pustych kolekcji z nowszych wersji javy i wykorzystuj je.

Reply:
https://programming.guide/java/emptylist-vs-new-collection.html

Reply:
z twoich uwag wydaje mi się że najlepsza będzie:Collections.emptyList() dzięki za uwagę :)